lib/presentation/orders_route.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:flutter/material.dart';
+
+class OrdersRoute extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
```

Comment:
Only root widget should be wrapped into Scaffold and is not required to have nested Scaffolds
https://api.flutter.dev/flutter/material/Scaffold-class.html (check Troubleshooting section)

Reply:
+ add listener for the app bat title as in the above sample

Reply:
Can you please show me that sample? Also, is it related to the bottom navigation task, should we made a separate task for Action Bar setup? Or even each of the screens should handle it by itself?

Reply:
sample can be found under
Troubleshooting
Nested Scaffolds
very first sample

Reply:
but yes, that's sample for AppBar. let's create another task for it.